FRAuth/FRAuth/Manager/TokenManager.swift

```diff
@@ -209,9 +192,9 @@ struct TokenManager {
     /// - Parameter completion: Completion block which will return an Error if there was any error encountered
     func revoke(completion: @escaping CompletionCallback) {
         do {
-            if let token = try self.retrieveAccessTokenFromKeychain() {
+            if let token = try self.keychainManager.getAccessToken() {
                 self.oAuth2Client.revoke(accessToken: token, completion: completion)
-                try? self.sessionManager.setAccessToken(token: nil)
+                try? self.keychainManager.setAccessToken(token: nil)
```

Comment:
Since this is already in a do catch block is there a reason for that to not throw and use the try?

Reply:
Updated as per previous comment.